ark/utils/deepcell_service_utils.py

```diff
@@ -141,12 +138,19 @@ def zip_write(zip_path):
 
         with ZipFile(zip_files[-1], "r") as zipObj:
             for name in zipObj.namelist():
+                # generate the path to save the segmentation mask
                 mask_path = os.path.join(deepcell_output_dir, name)
+
+                # DeepCell uses .tif extension, append extra f to account for .tiff standard
+                mask_path += 'f'
```

Comment:
Not a huge issue to fix this, but is there a reason DeepCell uses `.tif` instead of `.tiff`?

Reply:
You can talk to @srivarra, some parts of this are the default output from the zip object, some we control, I don't remember which is why

Reply:
@ngreenwald just talked with @srivarra, this is the default output from the `.zip` object (not something we control).